docs/running_docker.rst

```diff
@@ -41,20 +41,20 @@ Running MetaCI In Docker
 
 Below are the following steps necessary to run MetaCI on Docker:
 
-1. `.env File Creation and Variable Declaration`_
-    __ `.env File Creation and Variable Declaration`
+1. Configure MetaCI
 
-2. `Building Your Docker Containers`_ 
-    __ `Building Your Docker Containers`
+   Copy ``.env.example`` to create a file named ``.env`` in the top level of the MetaCI repository.
```

Comment:
Part of the problem is probably that I forgot to add `.env.example` to my commit. I'll do that now...